sdk/python/kfp/dsl/_container_op.py

```diff
@@ -17,10 +17,69 @@
 from . import _pipeline_param
 import re
 from typing import Dict
-
+from . import _types
+
+
+ComponentJsonSchema = '''
```

Comment:
@neuromage Does this look good to you?

Reply:
BTW, I also added the python native classes above.

Reply:
Yeah, I think this looks good. The type itself is unconstrained right now, but that's ok for now I think. The upper level structure looks great to me. It's a good starting point.

One question, we need to resolve it the name. Is it the Argo param name? If not, how do we resolve it to the Argo YAML? 
/cc @vicaire